site/_posts/2021-08-18-annnoucing-fusionauth-1.30.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog-post
+title: Announcing FusionAuth 1.30.0
+description: The FusionAuth 1.30.0 releases a robust suite of Advanced Threat Detection features, a JWT vending machine, myriad webhook events, and more.
+author: Akira Brand
+image: blogs/release-1-30/product-update-fusionauth-1.30.png
+category: blog
+tags: topic-troubleshooting
+excerpt_separator: "<!--more-->"
+---
+
+We're excited to announce the release of version 1.30.0 of FusionAuth. This version shipped on August 12, 2021. 1.30.0 includes a robust Advanced Threat Detection feature, adds 38 webhook events, resolves issues for FusionAuth community members and customers, and more! 
+
+<!--more-->
+
+This release contained a number of features, enhancements, and bug fixes. Please see the [release notes](/docs/v1/tech/release-notes/#version-1-30-0) for a full breakdown of the changes between 1.29 and 1.30.0. 
+
+There are a few items worth calling out.
+
+## JWT Vending Machine
+
+There are scenarios where a user does not yet exist, or is in the process of being created and needs access to services. This may be an anonymous user, or a future user. In these cases we cannot yet authenticate to FusionAuth because the user does not yet exist, or they may not yet have a password.
+
+To solve this, we have created the possiblity of opening an API that allows a JWT to be created with a payload defined by the API called.  This would require an API key, so the caller must be privledged.  
+
+Here is an example of the endpoint at work: 
+
+Endpoint: /api/jwt
+
+```
+
+{
+  "claims": {
+     "roles": [ "awesome" ],
+     "id": 42
+   }
+}
+```
+
+Using this ability, a new user on your app can start interacting with the app as a unique user, before needing to create an account. Then, once they do create an account, their unique Id is translated onto their profile. 
+
+## Threat Detection Feature
+
+Flagging and responding to suspicious behaviour is a huge advantage in any cybersecurity product, and we wanted to bring our customers as many advantages as we could in this arena. So, we created a large feature that has best-practice funtionality built in to deal with any sort of bizarre behaviours around user creation, user updates, and IP location and fingerprinting. 
```

Comment:
Preference: 
Flagging and responding to suspicious behavior is a huge advantage in any cybersecurity product, so naturally, we incorporated this functionality into the ever-evolving FusionAuth platform.

Reply:
So, we created a large feature that has best-practice functionality built in to deal with any sort of bizarre behaviors around user creation, user updates, and IP location and fingerprinting.

This sentence seems vague.
_maybe_?
In this release of FusionAuth, we use IP location data, user fingerprinting, and other indicators around user events to signal suspicious behavior.